src/cljam/io/vcf/util.clj

```diff
@@ -233,3 +234,54 @@
           (update :info stringify-info)
           (update fmt-kw stringify-format)
           (merge (into {} (for [k sample-kws] [k (stringify-sample (v fmt-kw) (v k))])))))))
+
+(def ^:private long-breakend-regexp
+  ;;     pre-seq      [ or ]  chr      pos   [ or ]    post-seq
+  #"((?:[ACGTN]*|\.))([\[\]])(.+?)(?::(\d+))([\[\]])((?:[ACGTN]*|\.))")
```

Comment:
This regexp is capturing `pre-seq` and `post-seq` after non-capturing, it can be written the following directly.

```clojure
;; pre-seq      [ or ]  chr      pos    [ or ]  post-seq
#"([ACGTN]*|\.)([\[\]])(.+?)(?::(\d+))?([\[\]])([ACGTN]*|\.)"
```

Reply:
Fixed in 51ac859. Thanks!